index.d.ts

```diff
@@ -260,13 +260,23 @@ export type Options<EncodingType = string> = {
 	Write some input to the `stdin` of your binary.
 	*/
 	readonly input?: string | Buffer | ReadableStream;
+
+	/**
+	Use a file as input to the the `stdin` of your binary.
```

Comment:
This should mention `input` somehow. Maybe using a `@see` annotation.

Reply:
I added a Tip like the `input` option. Would that work?